XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizard_UpgradeMode.resx

```diff
@@ -286,15 +292,114 @@
     <value>label2</value>
   </data>
   <data name="&gt;&gt;label2.Type" xml:space="preserve">
-    <value>System.Windows.Forms.Label, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    <value>System.Windows.Forms.Label, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </data>
   <data name="&gt;&gt;label2.Parent" xml:space="preserve">
-    <value>$this</value>
+    <value>tableLayoutPanel1</value>
   </data>
   <data name="&gt;&gt;label2.ZOrder" xml:space="preserve">
+    <value>7</value>
+  </data>
+  <data name="tableLayoutPanel1.ColumnCount" type="System.Int32, mscorlib">
+    <value>1</value>
+  </data>
+  <data name="applyUpdatesLabel.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="applyUpdatesLabel.Dock" type="System.Windows.Forms.DockStyle, System.Windows.Forms">
+    <value>Top</value>
+  </data>
+  <data name="applyUpdatesLabel.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
+    <value>NoControl</value>
+  </data>
+  <data name="applyUpdatesLabel.Location" type="System.Drawing.Point, System.Drawing">
+    <value>3, 238</value>
+  </data>
+  <data name="applyUpdatesLabel.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>3, 20, 3, 5</value>
+  </data>
+  <data name="applyUpdatesLabel.Size" type="System.Drawing.Size, System.Drawing">
+    <value>556, 26</value>
+  </data>
+  <data name="applyUpdatesLabel.TabIndex" type="System.Int32, mscorlib">
+    <value>8</value>
+  </data>
+  <data name="applyUpdatesLabel.Text" xml:space="preserve">
+    <value>Choose whether you want [XenCenter] to automatically download and install all the released updates after the selected pools are upgraded. Please note that internet connection is required for this option.</value>
+  </data>
+  <data name="&gt;&gt;applyUpdatesLabel.Name" xml:space="preserve">
+    <value>applyUpdatesLabel</value>
+  </data>
+  <data name="&gt;&gt;applyUpdatesLabel.Type" xml:space="preserve">
+    <value>System.Windows.Forms.Label, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </data>
+  <data name="&gt;&gt;applyUpdatesLabel.Parent" xml:space="preserve">
+    <value>tableLayoutPanel1</value>
+  </data>
+  <data name="&gt;&gt;applyUpdatesLabel.ZOrder" xml:space="preserve">
+    <value>0</value>
+  </data>
+  <data name="applyUpdatesCheckBox.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="applyUpdatesCheckBox.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
+    <value>NoControl</value>
+  </data>
+  <data name="applyUpdatesCheckBox.Location" type="System.Drawing.Point, System.Drawing">
+    <value>6, 274</value>
+  </data>
+  <data name="applyUpdatesCheckBox.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>6, 5, 3, 3</value>

```

Comment:
I think I would reset the margins to the default `3,3,3,3` so the checkbox is closer to the label above it, and its left edge is aligned with the left edge of the radio buttons (which do hang slightly outside the text of the labels).